File: Atividade_py/JoãoCiclano.py
taxa_juros = float(input("digite a taxa de juros anual: ")) / 100

valor_futuro = 14000 
anos = 12
valor_presente = valor_futuro / (1 + taxa_juros) ** anos

print(f"joão ciclano precisa depositar hoje R$ {valor_presente:.2f} para ter R$ 14000 em 12 anos.")

File: Atividade_js/JoãoCiclano.js
function calcularValorPresente(fv, taxa, anos) {
    return fv / Math.pow(1 + taxa, anos);
}

const taxajuros = 0.08;

const valor_futuro = 14000;
const anosJoaoCiclano = 12;
const valor_depositar = calcularValorPresente(valor_futuro, taxajuros, anosJoaoCiclano);

console.log(`joão ciclano precisa depositar hoje: $ ${valor_depositar.toFixed(2)}`);